src/pages/CartPage/CartPage.tsx

```diff
@@ -1,14 +1,16 @@
 import { StyledPage } from "../../common/styles/componentsStyles";
 
 import Hero from "../../components/Layout/Hero/Hero";
-import Cart from "../../components/Cart/CartList";
+import CartList from "../../components/Cart/CartList";
 
 const CartPage = () => {
   return (
-    <StyledPage>
-      <Hero />
-      <Cart />
-    </StyledPage>
+    <div style={{ height: "100vh" }}>
```

Comment:
It's better to not use inline styles. It's usually for "a quick test"

Reply:
This is what Helen wrote to test yesterday :)